backend/pages.py

```diff
@@ -13,5 +13,5 @@
     'game_event_source': 'backend.events:start_sse_stream',
     'join_game': 'backend.join_game:join_game',
     'increment_turn': 'backend.increment_turn:increment_turn',
-    'charge_rent': 'backend.charge_rent:charge_rent',
+    'check_position': 'backend.check_position:check_position',
```

Comment:
Does `check_position` need to be a page? I don’t think we’ll be calling it from the client.

Reply:
What I'm thinking is that `check_position` would be called every time a player has moved. When I think about it though, the server knows when a player has moved so it could call this module.

I'm not sure where it would be called from, though? Maybe `player_roll_dice` because that's where the server increments the players position in the database

Reply:
Yeah, I’d say we could just call it from the page for rolling.

Reply:
What I'm thinking is that `check_position` would be called every time a player has moved. When I think about it though, the server knows when a player has moved so it could call this module.

I'm not sure where it would be called from, though?